runners/samza/src/main/java/org/apache/beam/runners/samza/runtime/SamzaMetricsBundleProgressHandler.java

```diff
@@ -138,7 +138,7 @@ private void parseAndUpdateMetric(MetricsApi.MonitoringInfo monitoringInfo) {
         break;
 
       default:
-        LOG.warn("Unsupported metric type {}", monitoringInfo.getType());
+        LOG.debug("Unsupported metric type {}", monitoringInfo.getType());
```

Comment:
check if log.isDebugEnabled()?

Reply:
What do you want to do else?
If it is not enabled, then this line won't log. that is what we want right?

Reply:
Oh, right, I got into this habbit of checking debugEnabled to prevent debug log string is expensive to construct. Here is pretty straightfoward.

Reply:
Fixed!